libsolidity/formal/SMTEncoder.cpp

```diff
@@ -634,6 +634,10 @@ void SMTEncoder::endVisit(FunctionCall const& _funCall)
 	case FunctionType::Kind::Require:
 		visitRequire(_funCall);
 		break;
+	case FunctionType::Kind::Revert:
+		// Revert is a special case of require and equals to `require(false)`
+		addPathImpliedExpression(smtutil::Expression(false));
```

Comment:
Need to check if the string argument is visited for both require and revert, that part may use functions producing side-effects.

Reply:
Done.